hrjobcontract/js/test/controllers/contract_test.js

```diff
@@ -1,22 +1,57 @@
 define([
+    'common/moment',
     'common/angularMocks',
-    'job-contract/app'
+    'job-contract/app',
+    'job-contract/controllers/contract'
 ], function () {
     'use strict';
 
     describe('ContractCtrl', function () {
-        var ctrl;
+        var ctrl, $rootScope, $scope, $controller;
 
         beforeEach(module('hrjc'));
-        beforeEach(inject(function ($controller, $rootScope) {
-            var $scope = $rootScope.$new();
-            $scope.contract = { id: '1' };
+        beforeEach(inject(function (_$controller_, _$rootScope_) {
+
+          $controller = _$controller_;
+          $rootScope = _$rootScope_;
+
+          $scope = $rootScope.$new();
+
+          $scope.contract = {
+            id: '1',
+            contact_id: "84",
+            deleted: "0",
+            is_current: "0",
+            is_primary: "1"
+          };
+
+          $scope.$parent.contract = {
+            id: '1',
+            contact_id: "84",
+            deleted: "0",
+            is_current: "0",
+            is_primary: "1"
+          };
+
+          $scope.$parent.contractCurrent = [];
+          $scope.$parent.contractPast = [];
+
+          ctrl = $controller('ContractCtrl', {
+            $scope: $scope
+          });
 
-            ctrl = $controller('ContractCtrl', { $scope: $scope });
         }));
 
-        it('example', function () {
-            expect(true).toBe(true);
+        it('should make current contract', function () {
+          var d = new Date().toISOString().split("T")[0];
+          $scope.updateContractList(d);
+          expect($scope.$parent.contract.is_current).toBe("1");
+        });
+
+        it('should make past contract', function () {
```

Comment:
You shouldn't use "should" in the `it` descriptions, you should use the imperative mood
```js
it('makes past contract', function () {

});
```